logging/loggingtest/logger.go

```diff
@@ -96,8 +97,7 @@ func New() *Logger {
 	mute := make(chan bool)
 	quit := make(chan struct{})
 
-	// start muted to reduce spam during failing tests
-	muted := true
+	muted := !testing.Verbose()
```

Comment:
Within #1825 I've observed the problem that was not apparent unless testlogger is unmuted.